kalite/updates/views.py

```diff
@@ -2,7 +2,7 @@
 import os
 from annoying.decorators import render_to
 
-from django.utils.translation import ugettext_lazy
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Correct fix I think is acutally just reverting this line to:
`from django.utils.translation import ugettext as _, ugettext_lazy`